cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java

```diff
@@ -33,7 +35,14 @@ public static void main(String[] args) {
 
     initialProjectVersion();
 
-    Cli.CliBuilder<Runnable> builder = Cli.<Runnable>builder("java -jar toolkit-cli-" + projectVersion + ".jar");
+    String scriptName = System.getProperty("script.name");
+    Cli.CliBuilder<Runnable> builder = null;
+    if (StringUtils.isNotEmpty(scriptName)) {
+      builder = Cli.builder(scriptName);
+    } else {
+      builder = Cli.builder("java -jar toolkit-cli-" + projectVersion + ".jar");
+    }
```

Comment:
@kakulisen Yeah, it's better now. One more thing, the executable jar actually built is named 'cli-{version}.jar' not 'toolkit-cli-{version}.jar'